src/components/Search/beta/SearchDropdown.tsx

```diff
@@ -30,6 +30,7 @@ function SearchDropdown(props: SearchDropdownProps) {
   const [repoSelect, setRepoSelect] = useState({
     baseRepository: '',
     newRepository: '',
+    searchType: base,
```

Comment:
```suggestion
    searchType: 'base',
```

Reply:
Not in this case, however. Because I need to distinguish between 'base' and 'new' dropdowns.